examples/one-2-one/.yo-rc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We don't have `.yo-rc.json` in other example packages, can you please remove this file too?

Reply:
ok